load-balancing/elb-v2/common_functions.sh

```diff
@@ -374,15 +380,23 @@ wait_for_state() {
         return 1
     fi
 
-    msg "Checking $WAITER_ATTEMPTS times, every $waiter_interval seconds, for instance $instance_id to be in state $state_name"
+    # Check if a custom waiter_attempts was passed into the function
+    # and override the attemps if true
+    if [ -n ${var+5} ]; then
+        local waiter_attempts=$5
```

Comment:
Hmm... to me that's very confusing as I'm not used to bash's way of doing that. Can you instead grab all the arguments in the beginning of the script / method? Then here refer to the variable instead?